loadScript.js

```diff
@@ -1,13 +1,19 @@
 module.exports = function loadScript (src, cb) {
   var doc = document
-  var tag = 'script'
-  var firstScript
-  var el
-  el = doc.createElement(tag)
-  firstScript = doc.getElementsByTagName(tag)[0]
-  el.async = 1
-  el.src = src
+  var isCss = src.match(/\.css$/)
```

Comment:
I think this is dangerous, since webpack may rename the filename. I'd prefer to export a loadScript & loadStyle or better yet have a `tiny-style-loader`.